### ISSUE TITLE
SQLite database

### ISSUE THREAD

**Issue by alanmilinovic** [lKinderBueno/Streamity-Xtream-IPTV-Web-player#72]
If someone is developing this project, would be a good idea to support very light SQLite database as an alternative to Mysql.

**Comment by dillardblom** [comment]
I agree, it would make it much more portable and since (on average) only one user is filling epg at a time it wouldn't be very touchy (as SQlite can be acting strange if multiple users write to the database simultaneously). 

But, although I think this code looks promising (it even supports replay/catchup) while the codebase is nearly untouched for the past three years, i don't expect the author to pick up this challenge. Perhaps somebody can fork it (with enough knowledge on javascript)?
